gtda/plots/diagrams.py

```diff
@@ -0,0 +1,206 @@
+"""Persistence diagrams-retated plotting functions """
```

Comment:
I feel like the relation to persistence diagrams is stronger in `plots/homology.py` than here. For instance, one would think that a module containing "persistence diagrams-retated plotting functions" would contain utilities for plotting persistence diagrams themselves, but these are instead to be found in `plots/homology.py` .

Since we seem to be going for reflecting the structure of `gtda` here, should we go for something closer to the description of `gtda.diagrams`, which currently reads:
> The module gtda.diagrams implements transformers to preprocess persistence diagrams or extract features from them.

? E.g. we could say something like
```
"""Plotting functions for featurizations or representations of persistence
diagrams, coming in particular from the outputs of some classes in 
:mod:`gtda.diagrams`."""
```